docs/src/app/(public)/careers/design-engineer/page.tsx

```diff
@@ -0,0 +1,221 @@
+          </li>
+          <li className="mb-0.5">
+            Expert knowledge of CSS, including familiarity with common CSS tooling,
+            and knowledge of bleeding-edge CSS features.
+          </li>
+          <li className="mb-0.5">
+            Deep familiarity with the headless UI ecosystem. Strong opinions on the
+            pros and cons of API design choices across headless UI libraries.
+          </li>
+          <li className="mb-0.5">
+            Expert knowledge of a11y, including deep familiarity with ARIA
+            guidelines, WCAG success criterion, and screen reader technologies.
+          </li>
+          <li className="mb-0.5">Advanced understanding of design principles.</li>
+          <li className="mb-0.5">
+            A passion for craft, a keen eye for detail, and exquisite taste.
+          </li>
+          <li className="mb-0.5">
+            Excellent written and verbal communication skills.
+          </li>
+          <li className="mb-0.5">
+            Experience working remotely and communicating asynchronously.
+          </li>
+        </ul>
+        <h2 className="mt-10 mb-4 scroll-mt-6 text-xl font-medium text-balance">
+          Benefits
+        </h2>
+        <ul className="mb-4 ml-4.5 list-disc">
+          <li className="mb-0.5">$200k–$290k USD base salary.</li>
+          <li className="mb-0.5">
+            Subsidized healthcare package (dependent on employment path and
+            location).
+          </li>
+          <li className="mb-0.5">
+            Flexible time-off. We provide 33 days of paid time-off globally.
+          </li>
+          <li className="mb-0.5">
+            100% remote. Our entire company is globally distributed.
+          </li>
+          <li className="mb-0.5">
+            Company retreats. We meet up every 8 months for a week of work and fun.
+          </li>
+          <li className="mb-0.5">
+            Office equipment. We let you choose the hardware of your choice.
+          </li>
+          <li className="mb-0.5">
+            20% development time. Allocate 20% of your time towards personal and
+            professional development.
+          </li>
+          <li className="mb-0.5">
+            Education budget. We provide mentorship and send you to events that help
+            you build your network and skills.
+          </li>
+        </ul>
+        <p className="mb-4">
+          The actual salary will be determined by skill-level, experience, and
+          location. More information about the specific compensation package will be
+          shared during the hiring process.
+        </p>
+        <h2 className="mt-10 mb-4 scroll-mt-6 text-xl font-medium text-balance">
+          Application
+        </h2>
+        <p className="mb-4">The hiring process will consist of 5 stages:</p>
+        <ol className="mb-4 ml-4.5 list-inside list-decimal">
+          <li className="mb-0.5">Resume review.</li>
```

Comment:
```suggestion
          <li className="mb-0.5">Phone screeen</li>
```

We don't want to have everyone solve the tech challenge, lot of work for us to review, we can have initial pre-selection